src/site/layouts/faq_multiple_q_a.drupal.liquid

```diff
@@ -0,0 +1,84 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        {% if !entityPublished %}
+          <div class="usa-alert usa-alert-info" >
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+        <article class="usa-content">
+          <h1>{{ title }}</h1>
+
+          <div class="va-introtext">
+            {{ fieldIntroTextLimitedHtml.processed }}
+          </div>
+
+          {% if fieldAlertSingle %}
+            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% endif %}
+
+          {% if fieldButtonsRepeat %}
+            <div class="vads-u-margin-y--3">
+              {% for fieldButton in fieldButtons  %}
+                {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+              {% endfor %}
+            </div>
+          {% endif %}
+
+          {% if fieldTableOfContentsBoolean %}
+          <section id="table-of-contents">
+            <h2 class="vads-u-margin-bottom--2 vads-u-font-size--lg">On this page</h2>
+            <ul class="usa-unstyled-list"></ul>
```

Comment:
Is the content model not ready yet for this? Also should `On this page` be hardcoded or should it come from drupal?

Reply:
I think the content model is just `fieldTableOfContentsBoolean`. Then there is a build step that generates the table of contents. I just followed `page.drupal.liquid` as reference.